core/plugins/org.polarsys.capella.core.libraries.ui/src/org/polarsys/capella/core/libraries/ui/propertyTester/CapellaModelIsLocalPropertyTester.java

```diff
@@ -33,7 +35,12 @@ public boolean test(Object receiver_p, String property_p, Object[] args_p, Objec
       // the project is exported if there is no semantic model file at the same level of the aird file
       if ((selection != null) && (selection instanceof IFile)) {
         IFile airdFile = (IFile) selection;
-        return new IFileRequestor().search(airdFile.getParent(), CapellaResourceHelper.CAPELLA_MODEL_FILE_EXTENSION, false).size() > 0;
+        Session session = SessionManager.INSTANCE.getSession(URI.createURI(airdFile.getLocationURI().toString()), new NullProgressMonitor());
```

Comment:
The property tester shall be removed as the menu is now displayed in both cases, local and remote. 
Moreover, the getSession(uri) call will force to load the sirius session if we right click on the capella aird, even if it is closed. Session will be not opened but created (ie. loaded)
https://github.com/pcdavid/org.eclipse.sirius/blob/aee60e43454d04ff568b9ebd4e12f47d7661f19e/plugins/org.eclipse.sirius/src/org/eclipse/sirius/business/internal/session/SessionManagerImpl.java#L264-L282

Reply:
Oh right, I should have used getExistingSession here. 
I kept the property tester as a way to be sure that it only enable the menu if it is a project with a capella semantic model. Note that if I remove the property tester, for a modeling project without a capella semantic model, the menu is enabled but clicking on it does not open the Library dialog, so it does not lead to an issue, but it looks better to keep this property tester.